Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1031,6 +1054,30 @@ public extension Purchases {
 
 // swiftlint:enable missing_docs
 
+// MARK: - Paywalls
+
+@available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+public extension Purchases {
```

Comment:
man I wish we could use `package`

Reply:
We used to use objective-c headers added to the dependency for this kind of thing. I can't think of any clean way of doing this without having to expose the method.

Reply:
Yeah...